jarbas/dashboard/admin/__init__.py

```diff
@@ -120,7 +120,10 @@ def rosies_tweet(self, obj):
     def receipt_link(self, obj):
         if not obj.receipt_url:
             return ''
-        return mark_safe(f'<a target="_blank" href="{obj.receipt_url}">📃</a>')
+        image_src = '/static/image/receipt_icon.png'
```

Comment:
Codeclimate is complaining about the number of lines in this file. Perhaps you can create the image tag in only one line instead of 2:
```python
image_tag = '<img src="/static/image/receipt_icon.png">'
return mark_safe('<a target="_blank" href="{}">{}</a>'.format(obj.receipt_url, image))
```

Reply:
Fixed :+1: